src/FileHelper.php

```diff
@@ -149,10 +149,15 @@ public static function normalizePath(string $path): string
      *   Defaults to `false`, meaning the content of the symlinked directory would not be deleted.
      *   Only symlink would be removed in that default case.
      *
+     * - keepRootDirectory: boolean, whether only clear directory.
```

Comment:
Looks *very* inconsistent. A method is `removeDirectory` but the flag is "do not remove directory".

It is better to use `clearDirectory` in `removeDirectory`.

Reply:
Agree. I make this.